src/command_modules/azure-cli-vm/setup.py

```diff
@@ -37,6 +37,7 @@
     'azure-keyvault==0.3.7',
     'azure-mgmt-network==1.5.0rc3',
     'azure-mgmt-resource==1.2.0rc3',
+    'azure-mgmt-storage==1.2.0',
```

Comment:
The stack team is updating the dependency to 1.4.0

Reply:
Okay, let me close it and re-do it after stack team's gets in